build/props/common.props

```diff
@@ -6,30 +6,29 @@
         <LangVersion>preview</LangVersion>
         <Authors>.NET Foundation and Contributors</Authors>
         <PackageReleaseNotes>
-            Silk.NET May 2021 Update
+            Silk.NET June 2021 Update
 
-            - Add As methods to all Maths types &amp; other miscellaneous API additions
-            - Add support for provisional Vulkan Video encoding &amp; decoding extensions
-            - Update to latest OpenGL specification
-            - Update to latest OpenCL specification
-            - Update to Vulkan 1.2.177
-            - Update to OpenXR 1.0.15
-            - Fix ImGui robustness
-            - Fix disposal and reset robustness
-            - Fix SDL input backend not respecting deadzones
-            - Fix GLFW input backend not respecting start-up state
-            - Fix Vulkan device function pointer loader bugs
-            - Fix BSTR GlobalMemory issues
-            - Fix Vulkan codegen issues
-            - Fix bitfield codegen for Vulkan &amp; clang-generated bindings
-            - Miscellaneous windowing &amp; input bugfixes
+            - OpenXR bindings are now production-ready &amp; no longer experimental!
+            - Add (optional) native binaries for OpenAL Soft
+            - Add support for reversed-z projection matrices in Silk.NET.Maths
+            - Add bindings for a number of missing Vulkan and OpenCL extensions
+            - Add OpenGL-OpenXR interoperability helper APIs
+            - Add more helper APIs in Silk.NET.Core and Silk.NET.Maths
+            - Update to latest OpenGL and OpenGLES specifications
+            - Update to OpenXR 1.0.16
+            - Update to Vulkan 1.2.169
```

Comment:
This doesn't seem right?

Reply:
Was a typo for 179, latest commit regenerates and updates to 180.